Project1/include/Tile.h

```diff
@@ -8,21 +8,19 @@
 
 #include "Camera.h"
 
-class Tile
+class Tile: public Entity
 {
 public:
 	const static uint8_t WIDTH;
 	const static uint8_t HEIGHT;
 
 	Tile(std::string);
 
-	Point<uint64_t> Pos;
-	Point<uint16_t> Bounds;
-
-	std::string Texture;
-
 	bool Solid;
 	bool Portal;
 private:
+	void Update(){
+		return;
+	};

```

Comment:
Shouldn't need this function if it doesn't do anything. Don't make the Entity function a pure virtual function. https://stackoverflow.com/questions/2652198/difference-between-a-virtual-function-and-a-pure-virtual-function